local-setup/scripts/seed-notifications.py

```
@@ -0,0 +1,166 @@
+    for code in NOTIF_CODES:
+        print("  %-45s %s rows" % (code, count_rows(tok, code)))
+    # Core = Routing + Template (who + what). ProviderTemplate is the WhatsApp
+    # ContentSid layer (a follow-up); a failure there is a WARNING, not fatal to the
+    # install — the deploy still gets working config-driven SMS/Email notifications.
+    core_failed = [c for c in failed_masters if c != "RAINMAKER-PGR.NotificationProviderTemplate"]
+    note = ""
+    if failed_masters:
+        note = "  WARNING: failures in %s" % ", ".join(m.split(".")[-1] for m in failed_masters)
+    print("DONE: %d created, %d already-present.%s" % (total_created, total_dup, note))
+    sys.exit(2 if core_failed else 0)
+    sys.exit(0 if ok else 2)
```

Ruff (0.15.20)

[error] Undefined name `ok`

(F821)